OOP/assignment2/SurveyApplication.java

```diff
@@ -0,0 +1,204 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class SurveyApplication {
```

Comment:
I would appreciate if you use a better Object Oriented solution. Viz. All the inputs and dealt in one method, reports and dealt in some other method.